### PR TITLE
Fix adapter registration

### PR DESCRIPTION
**Pull request by gforcada**
I was trying to add some extra classes on the `<body>` and I expected to be an adapter to extend the functionality, and indeed, there it was: `plone.app.layout.globals.interfaces.IBodyClassAdapter` ✨ 

Even better, there is already an example adapter on `plone.app.layout.globals.layout.DefaultBodyClasses` 🤩 

There is one catch though, the adapter is not registered in ZCML (so that it gets a name) and is actually a multi adapter (context, request), but it was only registered as a single adapter.

Anyway, long story short: here is a fix for it 🎉 

I'm wondering if it would make sense to already split the `LayoutPolicy` (also in `p.a.l.g.layout`) and create smaller adapters? 🤔 

Anyway, meanwhile we can fix the actual code, so anyone else wanting to add some extra classes on the `<body>` class does not have to try to figure out how that works 😄

### CODE REVIEW

**Comment by yurj** [comment]
https://github.com/plone/documentation/blob/803d68e4fc87cd9990a9fa66a4496cd796af148e/docs/classic-ui/recipes.md?plain=1#L27

maybe the docs should be fixed then?

**Comment by gforcada** [comment]
Oh, nice, I thought there were no docs, I did not even do a search (my laziness, sorry 🤚🏾 ). But to me the docs look fine 🤔 but the ZCA is still a bit of a mystery for me 😓

**Review by jensens — APPROVED** [review]
This is a good catch. I only know one implementation which is the one in Mosaic somewhere which works fine.

**Review by mauritsvanrees — APPROVED** [review]
Partially I think this default adapter is only meant as an example, it does not actually return any classes.  But as example it indeed needs a fix, otherwise others copy it and it does not work.
